apps/stocks/src/app/app.component.ts

```diff
@@ -2,7 +2,6 @@ import { Component } from '@angular/core';
 
 @Component({
   selector: 'coding-challenge-root',
-  templateUrl: './app.component.html',
-  styleUrls: ['./app.component.css']
```

Comment:
removed css file reference since the css file was empty. In real case scenario it rarely happens to have a empty root component css file as empty.